build.gradle.kts

```
@@ -92,6 +95,7 @@ val copyGradleScripts by tasks.registering(Copy::class) {
     group = "build"
     description = "Copies the Gradle source and the generated scripts to the output directory."
 
+    val buildScanSummaryVersion = buildScanSummaryVersion
     // local variable required for configuration cache compatibility
     // https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:not_yet_implemented:accessing_top_level_at_execution
     val releaseVersion = releaseVersion
```

I recommend moving this below the comment since it's done for the same reason. Note there are several places where you can make the change. 

```suggestion
    // local variable required for configuration cache compatibility
    // https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:not_yet_implemented:accessing_top_level_at_execution
    val releaseVersion = releaseVersion
    val buildScanSummaryVersion = buildScanSummaryVersion
```

---

Done